README.md

```diff
@@ -103,34 +92,26 @@ To run the project, execute the following command:
 - Have more pages
 
 
-<!-- CONTRIBUTING -->
-
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
 > Write a message to encourage readers to support your project
 
 If you like this project...
```

Comment:
Kindly remove the description from the show support section and ensure the section is filled out as required 👍🏼